allensdk/brain_observatory/stimulus_analysis.py

```diff
@@ -410,7 +435,8 @@ def get_speed_tuning(self, binsize):
                 binned_cells_vis[:, i, 0] = np.mean(
                     celltraces_sorted_vis[:, :offset], axis=1)
                 binned_cells_vis[:, i, 1] = np.std(
-                    celltraces_sorted_vis[:, :offset], axis=1) / np.sqrt(offset)
+                    celltraces_sorted_vis[:, :offset], axis=1) / np.sqrt(
```

Comment:
can we put the linebreak after `:offset],` so that `np.sqrt(offset)` is all in one line?